File: Iniciante/URI 1044.go
package main
 
import (
    "fmt"
)
 
func main() {
    var a, b int

    fmt.Scanf("%d", &a)
    fmt.Scanf("%d", &b)

    if (a % b == 0 || b % a == 0) {
        fmt.Println("Sao Multiplos")
    } else { 
        fmt.Println("Nao sao Multiplos")
    }
}

File: Iniciante/URI 1044.clj
(require '[clojure.string :as str])

(def line (str/split (read-line) #" "))
(def A (Integer/parseInt (get line 0)))
(def B (Integer/parseInt (get line 1)))

(cond
    (or (== 0 (mod A B)) (== 0 (mod B A)))
        (println "Sao Multiplos")

    :else 
        (println "Nao sao Multiplos")   
)